### PR TITLE
updated sidebar copy for marionettejs example

### PR DESCRIPTION
**Pull request by kkemple**
updated copy for sidebar, removed wiki link and other out dated links, added links to more recent resources and updated description

this PR resolves https://github.com/marionettejs/marionettejs.com/issues/205

### CODE REVIEW

**Comment by samccone** [comment]
:clap: :+1:

File: learn.json [hunk 1]
```diff
@@ -1448,38 +1448,41 @@
...
 		"examples": [{
-			"name": "Example",
-			"url": "examples/backbone_marionette"
-		}],
+            "name": "Example Basic Marionette",
+            "url": "examples/backbone_marionette"
+        }, {
+            "name": "Example Marionette with RequireJS",
+            "url": "examples/backbone_marionette_require"
+        }],
```
Comment (stephenplusplus) [inline learn.json]: Tabs instead of spaces please :)

**Comment by kkemple** [comment]
fixed issue with spaces, and fixed incorrect url

File: learn.json [hunk 2]
```diff
@@ -1448,38 +1448,41 @@
...
+				"name": "MarionetteJS on GitHub",
 				"url": "https://github.com/marionettejs/backbone.marionette"
 			}]
 		}, {
 			"heading": "Articles and Guides",
 			"links": [{
-				"name": "A Thorough Introduction to Backbone.Marionette",
-				"url": "http://coding.smashingmagazine.com/2013/02/11/introduction-backbone-marionette"
+				"name": "Marionette 101",
+				"url": "https://www.youtube.com/watch?v=7yZKsgKxziw#t=106"
```
Comment (samccone) [inline learn.json]: ah remove `#t=106`

**Pull request merged**